src/test/java/org/jenkinsci/plugins/oic/ConfigurationAsCodeTest.java

```diff
@@ -1,70 +0,0 @@
-package org.jenkinsci.plugins.oic;
```

Comment:
I would recommend keeping casc tests working, so you don't break people who use the plugin with config as code.